### PR TITLE
fix: Plural texts  for updater UI

### PR DESCRIPTION
**Pull request by hofmeister**
Also fixes: 
- Issue when init'ing xterm
- init pendo on splash

### CODE REVIEW

File: src/renderer/components/plan-editor/updater/PlanUpdaterModal.tsx [hunk 1]
```diff
@@ -248,7 +252,18 @@ const PlanUpdaterNotice = (props: NoticeProps) => {
...
+                            There is <b>{props.planUpdater.updates.length}</b> update available for this plan
+                        </>
+                    )}
+                    {props.planUpdater.updates.length > 1 && (
+                        <>
+                            There are <b>{props.planUpdater.updates.length}</b> updates available for this plan
+                        </>
+                    )}
+
+                    {props.planUpdater.updates.length === 0 && <>There are no updates available for this plan</>}
```
Comment (andertun) [inline src/renderer/components/plan-editor/updater/PlanUpdaterModal.tsx:266]: Doesn't the conditional render above cover this case?

```
    if (planUpdater.updates.length === 0) {
        return null;
    }
```

**Review by andertun — COMMENTED** [review]
I share Jacobs concern with requiring pendo to load before we let the user in to the app. Maybe add a timeout instead? If pendo isn't ready after 10 seconds we consider it OK anyway.

**Comment by hofmeister** [comment]
fixed wrt pendo

**Review by wejendorp — APPROVED** [review]
👍

**Review by andertun — APPROVED** [review]

**Pull request merged**